src/icons/terminal_icon.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="21" height="18">
```

Comment:
nit: This should probably have a `viewBox` rather than an explicit width/height.